src/tests/Microsoft.PowerFx.Connectors.Tests/LivePublicSwaggerTests.cs

```diff
@@ -204,6 +197,25 @@ public async Task RealTest4()
             Assert.Equal("Africa/Abidjan", firstTZ);
         }
 
+        [Fact(Skip = "Live Test")]
+        public async Task RealTest5()
```

Comment:
>RealTest5 [](http://example.com/codeflow?start=26&length=9)

nit: Can we rename to something like, TestFailOnUnknownExtension()

Reply:
Done